abs.yaml

```
@@ -0,0 +1,2 @@
+upload_channels:
+  - sfe1ed40
```

```suggestion
  - sfe1ed40
  
channels:
  - https://staging.continuum.io/prefect/fs/setuptools-markdown-feedstock/pr1/750b3b0
   
```

An error on all platforms:
```
    conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("setuptools-markdown")}

    Encountered problems while solving:
      - nothing provides requested setuptools-markdown
```
As [setuptools-markdown 0.4.1](https://github.com/AnacondaRecipes/setuptools-markdown-feedstock/pull/1) is currently available on the staging channel, you need to add it to `abs.yaml ` for building on Prefect CI.
Staging channels are here https://staging.continuum.io/. Look at the last succeeded commit of setuptools-markdown's PR1.